group_vars/customer/eugene/main.yml

```diff
@@ -13,6 +13,10 @@ nextcloud_database_name: nextcloud
 elasticsearch_java_opts:
   - "-Xms128m"
   - "-Xmx128m"
+elasticsearch_app_resources:
```

Comment:
I didn't understand why Eugene needs to override this variable?

Reply:
Because 512MB (default value) is too much for a dev CI/env, it has been configured to use 128MB instead.

Reply:
I get the idea: Eugene is only for dev/CI so let's limit it.